lib/langchain/tool/news_retriever/news_retriever.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "type": "function",
+    "function": {
+      "name": "news_retriever__get_everything",
+      "description": "News Retriever: Search through millions of articles from over 150,000 large and small news sources and blogs.",
+      "parameters": {
+        "type": "object",
+        "properties": {
+          "q": {
+            "type": "string",
+            "description": "Keywords or phrases to search for in the article title and body. Surround phrases with quotes (\") for exact match. Alternatively you can use the AND / OR / NOT keywords, and optionally group these with parenthesis. Must be URL-encoded."
+          },
+          "search_in": {
+            "type": "string",
+            "description": "The fields to restrict your q search to.",
+            "enum": ["title", "description", "content"]
+          },
+          "sources": {
+            "type": "string",
+            "description": "A comma-seperated string of identifiers (maximum 20) for the news sources or blogs you want headlines from. Use the /sources endpoint to locate these programmatically or look at the sources index."
+          },
+          "domains": {
+            "type": "string",
+            "description": "A comma-seperated string of domains (eg bbc.co.uk, techcrunch.com, engadget.com) to restrict the search to."
+          },
+          "exclude_domains": {
+            "type": "string",
+            "description": "A comma-seperated string of domains (eg bbc.co.uk, techcrunch.com, engadget.com) to remove from the results."
+          },
+          "from": {
+            "type": "string",
+            "description": "A date and optional time for the oldest article allowed. This should be in ISO 8601 format."
+          },
+          "to": {
+            "type": "string",
+            "description": "A date and optional time for the newest article allowed. This should be in ISO 8601 format."
+          },
+          "language": {
+            "type": "string",
+            "description": "The 2-letter ISO-639-1 code of the language you want to get headlines for.",
+            "enum": ["ae", "ar", "at", "au", "be", "bg", "br", "ca", "ch", "cn", "co", "cu", "cz", "de", "eg", "fr", "gb", "gr", "hk", "hu", "id", "ie", "il", "in", "it", "jp", "kr", "lt", "lv", "ma", "mx", "my", "ng", "nl", "no", "nz", "ph", "pl", "pt"]
```

Comment:
Missing "en" for English, and the table I found says "eg" is Unassigned.

Reply:
I think you mistakenly put the parameters for get_top_headlines `country` here.  They should go below under `news_retriever__get_top_headlines`, and the `language` values listed in the docs should go here (https://newsapi.org/docs/endpoints/everything).

Reply:
@mattlindsey Nice find!